src/Init/Lean/PrettyPrinter/Parenthesizer.lean

```diff
@@ -325,8 +325,13 @@ else
 
 @[builtinParenthesizer termParser]
 def termParser.parenthesizer : Parenthesizer | p => visitAntiquot <|> do
-lbp ← evalNat p.appArg!;
-visitParenthesizable (fun stx => Unhygienic.run `(($stx))) lbp
+stx ← getCur;
+-- this can happen at `termParser <|> many1 commandParser` in `Term.stxQuot`
```

Comment:
This `<|>` is pretty tricky and requires the special case below to decide which side created a syntax tree. I would prefer to refactor the parser into two separate kinds `stxQuotTerm` and `stxQuotCommand`, but that's tricky without putting all of the former in a `try`.
```
def Term.stxQuotTerm := parser! symbol "`(" appPrec >> termParser >> ")"
def Term.stxQuotCommand := parser! symbol "`(" appPrec >> many1 commandParser true >> ")"
-- stxQuotCommand will never be used like this
@[builtinTermParser] def Term.stxQuot := Term.stxQuotTerm <|> Term.stxQuotCommand
```